addOns/ascanrules/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+### Changed
+- Depend on newer version of Common Library add-on.
+- Re-work Command Injection to handle timing utils changes.
```

Comment:
Remove this entry.

Reply:
Or if you want to be explicit about the fix, mention the scan rule in the fixed entry.

Reply:
Done